Scenes.md

```diff
@@ -126,16 +130,4 @@ class Level01 extends h2d.Scene {
         g.drawCircle( 20, -5, 3 );
     }
 }
-```
-
-## Scenes and camera
-
-Scenes and camera is explained here.
-
-### Sizing
-
-Scenes sizing is explained here.
-
-### Zoom
-
-Camera zoom is explained here.
```

Comment:
Removing these leftover "todos". We can always add information regarding zoom, sizing, etc. later.